changelog/@unreleased/pr-6778.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: migration
```

Comment:
This is only used in the tests, so I don't think it's a migration that Atlas users would need to be aware of - not sure we need a changelog for this actually

Reply:
Sure we can do with no changelog, since atlas tests will not work on ARM locally even with the updated image.

Reply:
Yep. Unfortunately putting no changelog didn't seem to remove this :( - I'm happy to remove this tomorrow if you want